Automattic/wp-calypso
# Onboarding: Update the image of Draft your first post modal

#### Changes proposed in this Pull Request

* Update the image of `DraftPostModal` and also adjust the height of action buttons

| Before | After |
| - | - |
| ![image](https://user-images.githubusercontent.com/13596067/137715040-532ad521-eaec-44a9-857b-a8389a3cdc94.png) | ![image](https://user-images.githubusercontent.com/13596067/137715198-e724f444-799f-49f4-9812-bf122ec4a1e8.png) |


#### Testing instructions



* Point your API to your sandbox
* Open another terminal and run
  * cd apps/wpcom-block-editor && yarn dev --sync
* Go to /start?flags=signup/hero-flow
* Finish the flow with `write` intent
* When you land on the editor, you should see the `write your first post modal` with latest image.



Related to https://github.com/Automattic/wp-calypso/issues/56568
